README.md

```diff
@@ -213,11 +213,11 @@ All props are detailed below.
 | `onCreateNode`             | `func`                     | `true`       | Called when a node is created.                                                                                                                                                              |
 | `onContextMenu`            | `func`                     | `true`       | Called when contextmenu event triggered.                                                                                                                                                              |
 | `onUpdateNode`             | `func`                     | `true`       | Called when a node is moved.                                                                                                                                                                |
-| `onDeleteNode`             | `func`                     | `true`       | Called when a node is deleted.                                                                                                                                                              |
+| `onDeleteNode`             | `func`                     | `true`       | Called when a node is deleted. This function should also delete connected edges. |
```

Comment:
Minor breaking change, we no longer call `onDeleteEdge` when we delete a node that has edges. The connected website should delete the extra edges itself. The reason is that calling `onDeleteEdge` for connected edges could cause the function to be called for the same edge if multiple connected nodes were deleted. This could cause errors if the edge was already deleted and the client code didn't account for that.

Please note that if you delete the node but not the edge then the edge will be considered orphaned. No errors will occur, but a console.warning will show to let the developer know that there's a problem. This warning could be useful for finding workflows with bad edge data.